docs/faq.md

```diff
@@ -12,6 +12,28 @@ hr {
 }
 </style>
 
+<hr id="operation-buffering-timed-out" />
+
+<a class="anchor" href="#operation-buffering-timed-out">**Q**</a>. Operation `...` timed out after 10000 ms. What gives?
+
+**A**. At its core, this issue stems from being unable to connect to mongoose. If you are using an application, like a bot,
+that wishes to connect to mongoose, you must ensure that you are connecting before the bot bot activates. For example, when working
```

Comment:
Can we do 1 sentence per line like in Mastering JS? Much easier for code review. I know the rest of the file doesn't follow this convention yet, but let's do 1 sentence per line for these new entries.